src/commands/downlink/GetShortName.ts

```diff
@@ -0,0 +1,92 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer, {IObis} from '../../CommandBinaryBuffer.js';
+import {DOWNLINK} from '../../constants/directions.js';
+
+
+/**
+ * IGetShortNameParameters command parameters
+ */
+interface IGetShortNameParameters {
+    obis: IObis
+}
+
+
+const COMMAND_ID = 0x01;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'get short name for OBIS code 8.0.9.1 - local time',
+        parameters: {
+            obis: {
+                groupA: 8,
+                groupC: 0,
+                groupD: 9,
+                groupE: 1
+            }
+        },
+        hex: {header: '01', body: '0a 08 00 09 01'}
+    }
+];
+
+
+/**
+ * Downlink command.
+ * TODO: rework example, it works only for current branch
+ *
+ * @example
+ * ```js
+ * import GetShortName from 'jooby-codec/commands/downlink/GetLmicInfo';
+ *
+ * const parameters = {
+ *     obis: {
+ *         groupA: 8,
+ *         groupC: 0,
+ *         groupD: 9,
+ *         groupE: 1
+ *     }
+ * };
+ * const command = new GetShortName(parameters);
+ *
+ * // output command binary in hex representation
+ * console.log(command.toHex());
+ * // 01 0a 08 00 09 01
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/obis-observer/commands/GetShortName.md#request)
+ */
+class GetShortName extends Command {
+    constructor ( public parameters: IGetShortNameParameters ) {
+        super();
+
+        this.size = CommandBinaryBuffer.getObisSize(parameters.obis);
```

Comment:
do we need this? should we add this to analog commands?

Reply:
yes, here - needed, we can add this to analog commands